### PR TITLE
Allow updating provisioning status

### LINKED ISSUE #33: Add callback for provisioning mechanics
When the namsapces are provisioned the registry should be provided feedback so it knows the status of provisioning (success / fail). This will allow the registry to mark the namespaces as "created" and, if any other workflows are needed to be triggered, do so.

### PR DESCRIPTION
**Pull request by jleach**
This change allows for the callback from the namespace provisioning mechanics to the registry API to update the provisioning status of said namespace.

Fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,3 +75,173 @@ psql -U postgres -d registry -h host.docker.internal -f /opt/src/db/sql/0001.sql
 Seeing an unresolved image or not deploying the API? Remember to tag:
 oc tag platsrv-registry-api:latest platsrv-registry-api:dev
 
+ult -n devops-security-aporeto-operator
```
Comment (ShellyXueHan) [inline README.md:78]: what are the logs for?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,20 @@
...
+export SSO_TOKEN_URL=YOUR_TOKEN_URL_HERE
+
+curl -sX POST -H "Content-Type: application/x-www-form-urlencoded" \
+    --data "grant_type=client_credentials&client_id=${SSO_CLIENT_ID}&client_secret=${SSO_TOKEN}" \
+    ${SSO_TOKEN_URL} | \
+    jq '.access_token' |
+    cut -d "\"" -f 2
+
+Then use a callback:
+curl -vX PUT -H "Content-Type: application/json" -H "Authorization: Bearer ${MYT}" http://localhost:8100/api/v1/provision/1/namespace -d @data.json
```
Comment (ShellyXueHan) [inline README.md:21]: instruction missing a step to set token value to var MYT

File: README.md (hunk 1 above)
Comment (jleach) [reply README.md:78]: Those are just notes in the README. I'm currently working through better documentation.

File: README.md (hunk 2 above)
Comment (jleach) [reply README.md:21]: Those are just notes in the README. I'm currently working through better documentation.

**Pull request merged**